test-requirements.txt

```diff
@@ -1 +1,2 @@
 mock
+unittest2
```

Comment:
unittest2 allows us to use newer unittest things in older python especially < 2.7 such as skipping test decorators